news/New Changes to Star Rating and PP

```diff
@@ -0,0 +1,38 @@
+Below is the PR for readability.
+
+"Hello!
+
+It is with anticipation that I, Xexxar, am here to announce changes to both the PP system as well as the star rating system. In fact, the star rating system, formally known as DiffCalc, has not received an update in several years, making this one of the most exciting updates to the game in a long time. The game has changed considerably since the last update to DiffCalc with the introduction of many new styles of maps and gameplay. While this change is not designed to fix every issue with the system, it's fundamental purpose is to better assess skills that may have been missing from previous iterations of the algorithm. Without further delay, let's get into the specifics.
+
+## Wide Angle Jumps
+
+As many players are well aware, the angle of a jump impacts how difficulty a pattern is to combo. Evidence of this can be found all throughout modern mapping. Almost all maps that are popular for their PP values contain jumps that are typically non-linear. As such, through the usage of a new angle calculation, a new scaling reward for jumps with angles exceeding 60 degrees has been added. Decided not to farm a map because it had square jumps? Found yourself baffled by the difficulty of an older 2013 map with linear flow? These previously underweighted maps now have more accurate star rating associated with them (this includes alt maps). And yes, they will be worth more PP as a result.
```

Comment:
```suggestion
As many players are well aware, the angle of a jump impacts how difficulty a pattern is to combo. Evidence of this can be found all throughout modern mapping. Almost all maps that are popular for their PP values contain jumps that are typically non-linear. As such, through the usage of a new angle calculation, a new scaling reward for jumps with angles exceeding 60 degrees has been added. Decided not to farm a map because it had square jumps? Found yourself baffled by the difficulty of an older 2013 map with linear flow? These previously underweighted maps now have a more accurate star rating associated with them (this includes alt maps). And yes, they will be worth more PP as a result.
```